### PR TITLE
Sequence annotations implementations starter code

### PR DESCRIPTION
**Pull request by macieksmuga**
Supersedes #857 and re-introduces a GFF3 parser, script to create SQLite3 files of sequence feature annotations from GFF3 source files, as well as a `datamodel.sequenceAnnotations` module to read rows from the resulting database files and convert them to GA4GH objects.

### CODE REVIEW

**Comment by macieksmuga** [comment]
I just removed the "(WIP)" tag on this PR, as it now represents a working (as best as I can tell) implementation of the Sequence Annotations API.

File: ga4gh/backend.py [hunk 1]
```diff
@@ -429,6 +429,17 @@ def variantsGenerator(self, request):
         intervalIterator = VariantsIntervalIterator(request, variantSet)
         return intervalIterator
 
+    def featuresGenerator(self, request):
+        """
+        Because, I guess, a generator is needed here?
+        :param request: the original web request, passed merrily along
+        :return: an iterator over the result set?
+        """
```
Comment (diekhans) [inline ga4gh/backend.py]: ;-) please provide real doc...

**Comment by diekhans** [comment]
Please don't check in binary files (tests/data/datasets/dataset1/sequenceAnnotations/sacCerTest.db), have the process build them from test source files.

**Comment by macieksmuga** [comment]
@diekhans I added the binary `.db` files for ease of testing, in parallel with a directory-full of binary BAM files (in `tests/data/datasets/dataset1/reads`). While I understand your point, I believe it's simpler to just keep these binaries there for now.

**Comment by diekhans** [comment]
Oh, we really should be checking BAMs in either.  But guess we
can live with it for now.

**Comment by macieksmuga** [comment]
I just rebased this PR (ie. the source branch fro this PR) on top of the latest ga4gh/server:master, which includes all the Variation Annotation code.

**Comment by diekhans** [comment]
This PR seems to have lots of unrelated commits in it.

**Comment by macieksmuga** [comment]
Closing this pull request due to a messy commit history. Reopening soon...

**Pull request closed without merging**